src/Row/index.js

```diff
@@ -55,7 +58,7 @@ Row.propTypes = {
   /** Controls vertical alignment of items within the row */
   alignItems: PropTypes.oneOf(["top", "center"]),
   /** Controls horizontal flex justification */
-  justifyContent: PropTypes.oneOf(["start", "end"]),
+  justifyContent: PropTypes.oneOf(["start", "end", "space-between"]),
```

Comment:
Seems useful when we place cancel/confirm actions at opposite sides of an element (like the drawer footer designs)